wbooks-api/spec/factories/rents.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :rent do
+    loan            { Faker::Date.backward(5) }
+    restitution     { Faker::Date.forward(5) }
+    association     :user, id: 1, strategy: :build
```

Comment:
if you change this line for `user` will automatically create an associated user for the rent

Reply:
nice!

Reply:
Agreed with this!